content/it/cloud_native_apps.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Cloud Native Apps
+status: Completed
+category: Concetto
+---
+
+## Cos'è
+Le applicazioni Cloud Native sono progettate specificamente per sfruttare le innovazioni nel [Cloud Computing](it/cloud_computing/). Queste applicazioni si integrano facilmente con le rispettive architetture cloud sfruttandone le risorse e le capacità in termini di [scalabilità](it/scalability/). La definizione si riferisce anche ad applicazioni che sfruttano le innovazioni in ambito infrastrutturale, introdotte dal Cloud Computing. Con applicazioni Cloud Native oggi si intendono app che vengono eseguite indifferentemente su data center di un provider cloud o su piattaforme Cloud Native on-premise.
+
+## Quali problematiche risolve
+I tradizionali ambienti on-premise fornivano risorse di calcolo in modo abbastanza personalizzato. Ogni data center disponeva di servizi basati su [architetture_con_un forte_accoppiamento](/tightly_coupled_architectures/) tra applicazioni e ambiente specifico, con un pesante provisioning manuale dell'infrastruttura, come [macchine virtuali](it/virtual_machine/) e servizi. Questo, a sua volta, ha vincolato gli sviluppatori e le loro applicazioni a quel data center specifico. Ma le applicazioni non progettate per il cloud non potevano sfruttarne le capacità di resilienza e scalabilità (pensiamo, per esempio, alle app che richiedendo un intervento manuale per avviarsi correttamente non possono essere ridimensionate automaticamente, né possono essere riavviate automaticamente in caso di errore).
```

Comment:
```suggestion
I tradizionali ambienti on-premise fornivano risorse di calcolo in modo abbastanza personalizzato. Ogni data center disponeva di servizi basati su [architetture_con_un forte_accoppiamento](/it/tightly_coupled_architectures/) tra applicazioni e ambiente specifico, con un pesante provisioning manuale dell'infrastruttura, come [macchine virtuali](/it/virtual_machine/) e servizi. Questo, a sua volta, ha vincolato gli sviluppatori e le loro applicazioni a quel data center specifico. Ma le applicazioni non progettate per il cloud non potevano sfruttarne le capacità di resilienza e scalabilità (pensiamo, per esempio, alle app che richiedendo un intervento manuale per avviarsi correttamente non possono essere ridimensionate automaticamente, né possono essere riavviate automaticamente in caso di errore).
```

Reply:
@fsbaraglia quindi /it/ anche se il termine non è ancora stato localizzato? 👍🏻 alles klar